### ISSUE TITLE
Promote the project

### ISSUE THREAD

**Issue by naXa777** [mdht/mdht#158]
Guys, you do a great job! your tools are really helpful. But I think this repository requires more activity. Participate in the ongoing Hacktoberfest to attract collaborators for example.

**Comment by howlingsails** [comment]
Who's using this in some way? I bet a lot of people... how many downloads?  
No specifics... :-)

;-)
 I'll guess it has to do with health care!

**Comment by swmuir** [comment]
Yes this project has to do with health care - the core project has moved to eclipse 
https://projects.eclipse.org/projects/modeling.mdht
i thought that there was a notice on this repo but will look again
the only item still active is our publishing support for dita as we are not allowed to have that content on the eclipse site due to some licenses

**Comment by howlingsails** [comment]
What is the mdht-model project and how does it relate to this repo?

**Comment by swmuir** [comment]
the projects in mdht-models are the actual implementation guide projects for various cda implementations
several are obe consol is the consolidated spec implementation used by validations services
the projects are broken up into model, documentation, generated - generated is the jar files for creating,consuming, and validating the consol cda documents
we have some examples i think they should be working but we have lost some maven repos so not sure
